modules/bmo-performing-a-live-update-to-the-hostfirmwarecomponents-resource.adoc

```diff
@@ -0,0 +1,79 @@
+// This is included in the following assemblies:
+//
+// * installing/installing_bare_metal/ipi/ipi-install-post-installation-configuration.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="bmo-performing-a-live-update-to-the-hostfirmwarecomponents-resource_{context}"]
+= Performing a live update to the HostFirmwareComponents resource
+
+You can perform a live update to the `HostFirmwareComponents` resource on an already provisioned host. Live updates do not trigger deprovisioning and reprovisioning the host. To update the `HostFirmwareComponents` resource, perform the following actions:
+
+* Edit or patch the `HostFirmwareComponents` resource.
+* Cordon and drain the node.
+* Power off the host for 5 minutes.
+* Power on the host.
+* Uncordon the node so it can rejoin the cluster.
```

Comment:
Suggestion: This summary can be skipped since it's a small procedure that is easy to read.